Lib/test/test_embed.py

```diff
@@ -471,6 +471,24 @@ class InitConfigTests(EmbeddingTestsMixin, unittest.TestCase):
             ('Py_LegacyWindowsStdioFlag', 'legacy_windows_stdio'),
         ))
 
+    # path config
+    DEFAULT_PATH_CONFIG = {}
+    if MS_WINDOWS:
+        DEFAULT_PATH_CONFIG.update(isolated=-1, site_import=-1)
+    COPY_PATH_CONFIG = [
+        # Copy core config to global config for expected values
+        #'program_full_path',
+        'prefix',
+        'exec_prefix',
+        #'module_search_path',
+        'program_name',
+        'home',
+    ]
+    if MS_WINDOWS:
```

Comment:
IIRC, base_executable applies on macOS too.

Reply:
But maybe only the sys module attribute is used and not the path config struct?

Reply:
The idea of this PR is to test _PyPathConfig content: the global _Py_path_config variable.

sys.base_executable is set from PyConfig.base_executable which is already tested in test_embed.